### PR TITLE
Add lua-resty-jwt to nginx image

### PR DESCRIPTION
**Pull request by lake-of-dreams**
## What this PR does / why we need it:
The nginx image built using this repository is going to be used for the api server for verrazzano which will perform JWT validation from the console UI. This requires the `lua-resty-jwt` lua module to be installed. Also few libraries are updated to mitigate reported security vulnerabilities.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?
Manually tested by using the image built after this change as api server

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/kubernetes/ingress-nginx/blob/master/CONTRIBUTING.md) guide
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: images/nginx/rootfs/Dockerfile [hunk 1]
```diff
@@ -56,6 +56,30 @@ RUN microdnf install  \
     nano \
     tzdata \
     pkg-config \
+  && microdnf install yum \
```
Comment (markxnelson) [inline images/nginx/rootfs/Dockerfile:59]: don't install yum, just use microdnf

**Review by markxnelson — CHANGES_REQUESTED** [review]
don't install yum, just use microdnf

File: images/nginx/rootfs/Dockerfile (hunk 1 above)
Comment (lake-of-dreams) [reply images/nginx/rootfs/Dockerfile:59]: @markxnelson  microdnf does not seem to be installing latest packages, for example
1/ Before introducing `yum` , microdnf was already installing libxml2. - see https://github.com/verrazzano/ingress-nginx/blob/oracle-build-from-source/images/nginx/rootfs/Dockerfile#L54 . This installs `libxml2-devel-2.9.7-7.0.1.el8.x86_64`.
2/ The scanning report https://build.verrazzano.io/view/oracle-build-from-source/job/ingress-nginx/job/bfs%252Fvz1769/1/artifact/scanning-report.json shows a vulnerability with this version.
3/ If we do microdnf update, nothing happens.
4/ After introducing `yum` , libxml2 is updated to libxml2-2.9.7-8.0.1.el8.x86_64 which is the fixed version.

Please suggest.

Comment (lake-of-dreams) [reply images/nginx/rootfs/Dockerfile:59]: One thing I can try is specifying the version with `microdnf install` - will try that

Comment (lake-of-dreams) [reply images/nginx/rootfs/Dockerfile]: fixed.

**Comment by lake-of-dreams** [comment]
> don't install yum, just use microdnf

implemented

**Review by markxnelson — APPROVED** [review]

**Pull request merged**